### PR TITLE
[FEATURE] #103529 - Introduce hotkey for "Save and Close"

### PR DESCRIPTION
**Pull request by sfinnerWco**
res: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/887
Releases: main

### CODE REVIEW

**Review by garvinhicking — APPROVED** [review]

**Comment by garvinhicking** [comment]
Do you want to work more on it, as it's a draft? I think it is an improvement even now :) Thanks!!

File: Documentation/HelpInside/KeyboardCommands/Index.rst [hunk 1]
```diff
@@ -23,6 +23,20 @@ the guidelines set out in `WAI-ARIA Authoring Practices 1.1. <https://www.w3.org
 *  :kbd:`Right` arrow key will expand the focus if possible.
 *  :kbd:`Left` arrow key will close the focus if possible.
 
+Actions in backend editor
```
Comment (garvinhicking) [inline Documentation/HelpInside/KeyboardCommands/Index.rst:26]: How about "in backend forms"? Maybe "editor" might make people think it's related to the RTE?

**Comment by sfinnerWco** [comment]
I put it in as a draft cause I wanted to also reference the keyboard commands page inside different Subpages of the "Content Elements" Page - for letting users know better that there are shortcuts in the editor.

According to your suggestion to name ist "Actions in backend forms" I agree, I will change that!

File: Documentation/ContentElements/Content/Index.rst [hunk 2]
```diff
@@ -65,3 +65,9 @@ Highlighted areas will then show you where you can drop it.
 
 You can move a content element from one page to another in the :guilabel:`Web
 > List` module using cut and paste. Refer to the :ref:`clipboard chapter <clipboard>` for information about using the clipboard in the :guilabel:`List` module.
+
+.. tip::
+
+   If you are more comfortable using shortcuts, there are different kinds of
+   :ref:`keyboard commands <t3editors:keyboard_commands>` you can use in
+   the Backend Editor.
```
Comment (garvinhicking) [inline Documentation/ContentElements/Content/Index.rst:73]: Maybe also use "backend forms" here, too?

File: Documentation/ContentElements/CreatingContent/Index.rst [hunk 3]
```diff
@@ -143,3 +143,9 @@ Add new content to a page
    :class: with-shadow
 
    The new content element appears in the Page module
+
+.. tip::
+
+   If you are more comfortable using shortcuts, there are different kinds of
+   :ref:`keyboard commands <t3editors:keyboard_commands>` you can use in
+   the Backend Editor.
```
Comment (garvinhicking) [inline Documentation/ContentElements/CreatingContent/Index.rst:151]: (Same as above)

File: Documentation/ContentElements/EditingContent/Index.rst [hunk 4]
```diff
@@ -26,3 +26,9 @@ display different input fields.
 
 This action does not delete content. It is saved in the background and will be
 restored if you select the applicable content element type again.
+
+.. tip::
+
+   If you are more comfortable using shortcuts, there are different kinds of
+   :ref:`keyboard commands <t3editors:keyboard_commands>` you can use in
+   the Backend Editor.
```
Comment (garvinhicking) [inline Documentation/ContentElements/EditingContent/Index.rst:34]: Seeing this happens multiple times,
how do you think about outsourcing this tip as a rst include file and doing an include of that at the current places?
(I can also do that in a follow-up, but if you'd be interested to do it, that would be a great help!)

File: Documentation/ContentElements/Content/Index.rst (hunk 2 above)
Comment (sfinnerWco) [reply Documentation/ContentElements/Content/Index.rst:73]: thanks for the hint. I will change it in this parts as well

File: Documentation/ContentElements/EditingContent/Index.rst (hunk 4 above)
Comment (sfinnerWco) [reply Documentation/ContentElements/EditingContent/Index.rst:34]: I could do that, yeah. Is there any convention for where to place that include file? Like in "Documentation/ContentElements/Includes" or just place it in the existing ContentElements Folder next to the Index.rst?

Comment (garvinhicking) [reply Documentation/ContentElements/EditingContent/Index.rst:34]: In this repo we don't have includes yet. In the Core API docs we often create subdirectories only when more than one file is included.

In your case here I would put a file like "TipHotkeys.rst" or so just into the same directory as Index.rst, yes. We can move it later when more includes happen.

Thanks! :)

**Comment by garvinhicking** [comment]
Let me know when you're ready on your side, then I'll render and merge this. 👍

**Comment by sfinnerWco** [comment]
I guess I have to rename the file to .rst.txt as an include (at least it's like this in the Core API docs).
Afterwards it's ready for me.

**Comment by sfinnerWco** [comment]
should be fine now I hope!

File: Documentation/ContentElements/TipKeyboardCommands.rst.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.. tip::
+
+   If you are more comfortable using shortcuts, there are different kinds of
+   :ref:`keyboard commands <keyboard_commands>` you can use in
+   the backend forms.
```
Comment (froemken) [inline Documentation/ContentElements/TipKeyboardCommands.rst.txt:5]:
```suggestion
..  tip::

    If you are more comfortable using shortcuts, there are different kinds of
    :ref:`keyboard commands <keyboard_commands>` you can use in
    the backend forms.
```

File: Documentation/HelpInside/KeyboardCommands/Index.rst [hunk 6]
```diff
@@ -23,6 +23,20 @@ the guidelines set out in `WAI-ARIA Authoring Practices 1.1. <https://www.w3.org
...
+
+    ..  group-tab:: Windows / Linux
+
+        *  :kbd:`ctrl` + :kbd:`s` = Save
+        *  :kbd:`ctrl` + :kbd:`shift` + :kbd:`s` = Save and close
+
+    ..  group-tab:: macOs
+
+        *  :kbd:`cmd (⌘)` + :kbd:`s` = Save
+        *  :kbd:`cmd (⌘)` + :kbd:`shift` + :kbd:`s` = Save and close
```
Comment (froemken) [inline Documentation/HelpInside/KeyboardCommands/Index.rst:39]:
```suggestion
    ..  group-tab:: Windows / Linux

        *   :kbd:`ctrl` + :kbd:`s` = Save
        *   :kbd:`ctrl` + :kbd:`shift` + :kbd:`s` = Save and close

    ..  group-tab:: macOs

        *   :kbd:`cmd (⌘)` + :kbd:`s` = Save
        *   :kbd:`cmd (⌘)` + :kbd:`shift` + :kbd:`s` = Save and close
```

**Comment by garvinhicking** [comment]
Many thanks for working on it! 👍

**Pull request merged**